datasets/dbrd/README.md

```diff
@@ -0,0 +1,170 @@
+
+### Source Data
+
+The book reviews were taken from [Hebban](https://www.hebban.nl), a Dutch platform for book reviews.
+
+#### Initial Data Collection and Normalization
+
+The source code of the scraper and preprocessing process can be found in the [DBRD GitHub repository](https://github.com/benjaminvdb/DBRD).
+
+#### Who are the source language producers?
+
+The reviews are written by users of [Hebban](https://www.hebban.nl) and are of varying quality. Some are short, others long, and many contain spelling mistakes and other errors. 
+
+### Annotations
+
+Each book review was accompanied by a 1 to 5 star rating. The annotations are produced by mapping the user-provided ratings to either positive or negative label. 1 and 2 star ratings are given the negative label `0` and 4 and 5 star ratings the positive label `1`. Reviews with a rating of 3 stars are considered neutral and left out of the `train`/`test` sets and added to the unsupervised set.
+
+#### Annotation process
+
+Users of [Hebban](https://www.hebban.nl) were unaware that their reviews would be used in the creation of this dataset.
+
+#### Who are the annotators?
+
+The annotators are the [Hebban](https://www.hebban.nl) users who wrote the book review associated with the annotation. Anyone can register on [Hebban](https://www.hebban.nl) and it's impossible to know the demographics of this group. 
+
+### Personal and Sensitive Information
+
+The book reviews and ratings are publicly available on [Hebban](https://www.hebban.nl) and no personal or otherwise sensitive information is contained in this dataset.
+
+## Additional Information
+
+### Dataset Curators
+
+This dataset was created by [Benjamin van der Burgh](mailto:benjaminvdb@gmail.com), who was working at [Leiden Institute of Advanced Computer Science (LIACS)](https://liacs.leidenuniv.nl/) at the time.
+
+### Licensing Information
+
+The dataset is licensed under a [Creative Commons Attribution-NonCommercial-ShareAlike 4.0 International License](https://creativecommons.org/licenses/by-nc-sa/4.0/).
+
+### Citation Information
+
+Please use the following citation when making use of this dataset in your work.
+
+```
+@article{DBLP:journals/corr/abs-1910-00896,
+  author    = {Benjamin van der Burgh and
+               Suzan Verberne},
+  title     = {The merits of Universal Language Model Fine-tuning for Small Datasets
+               - a case with Dutch book reviews},
+  journal   = {CoRR},
+  volume    = {abs/1910.00896},
+  year      = {2019},
+  url       = {http://arxiv.org/abs/1910.00896},
+  archivePrefix = {arXiv},
+  eprint    = {1910.00896},
+  timestamp = {Fri, 04 Oct 2019 12:28:06 +0200},
+  biburl    = {https://dblp.org/rec/journals/corr/abs-1910-00896.bib},
+  bibsource = {dblp computer science bibliography, https://dblp.org}
+}
+```
```

Comment:
Looks like the `Considerations for Using the Data` section has been removed

Could you re-add it please ? you can leave the content as `[More Information Needed]`

Reply:
I've re-added the `Considerations for Using the Data` section and provided two of its subsections. Only information `Discussion of Biases` is missing now and this is indicated with `[More Information Needed]`.